test/player-proxy-maker.js

```diff
@@ -3,26 +3,27 @@ import videojs from 'video.js';
 
 const proxy = (props) => {
   const player = extend(true, {}, videojs.EventTarget.prototype, {
-    play: Function.prototype,
-    paused: Function.prototype,
-    ended: Function.prototype,
-    poster: Function.prototype,
-    src: Function.prototype,
-    currentSrc: Function.prototype,
-    addRemoteTextTrack: Function.prototype,
-    removeRemoteTextTrack: Function.prototype,
-    remoteTextTracks: Function.prototype,
+    play: () => {},
```

Comment:
Removed uses of `Function.prototype` as a noop because it has caused issues in the past with Video.js' use of `guid` for tracking bindings.